vendor/directx/dxgi/dxgi.odin

```diff
@@ -481,8 +493,16 @@ IResource_VTable :: struct {
 	using idxgidevicesubobject_vtable: IDeviceSubObject_VTable,
 	GetSharedHandle:     proc "stdcall" (this: ^IResource, pSharedHandle: ^HANDLE) -> HRESULT,
 	GetUsage:            proc "stdcall" (this: ^IResource, pUsage: ^USAGE) -> HRESULT,
-	SetEvictionPriority: proc "stdcall" (this: ^IResource, EvictionPriority: u32) -> HRESULT,
-	GetEvictionPriority: proc "stdcall" (this: ^IResource, pEvictionPriority: ^u32) -> HRESULT,
+	SetEvictionPriority: proc "stdcall" (this: ^IResource, EvictionPriority: RESOURCE_PRIORITY_FLAG) -> HRESULT,
+	GetEvictionPriority: proc "stdcall" (this: ^IResource, pEvictionPriority: ^RESOURCE_PRIORITY_FLAG) -> HRESULT,
+}
+
+RESOURCE_PRIORITY_FLAG :: enum u32 { // TODO: convert to bit_set
```

Comment:
This one seems like an enum (one of the options), not a bitset (flags), maybe remove the `TODO` comment.